Doc/library/string.rst

```diff
@@ -571,6 +571,12 @@ The available presentation types for :class:`float` and
    |         | the current locale setting to insert the appropriate     |
    |         | number separator characters.                             |
    +---------+----------------------------------------------------------+
+   | ``'x'`` | For :class:`float`, hex format. Outputs the number in    |
+   |         | base 16, using lower-case letters for the digits above 9.|
+   |         | The alternate form adds ``'0x'`` prefix.                 |
+   |         |                                                          |
+   |         | Precision and ``'='`` alignment are not supported.       |
```

Comment:
Hmm, what's wrong with ``=``?  This seems to be supported on my branch:
```pycon
>>> f"{-1.1:30x}"
'           -1.199999999999ap+0'
>>> f"{-1.1:=30x}"
'-           1.199999999999ap+0'
>>> f"{-1.111111:=30f}"
'-                     1.111111'
>>> f"{-1.111111:30f}"
'                     -1.111111'
```

Reply:
I was just too lazy to implement it. If it matters, I can change my implementation to support the "=" alignment as well.